<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QMainWindow" name="QtWidgetsApplication1Class">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="slectefile">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>fichier</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confDcodage">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>lancer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAffiche">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>340</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PORtlineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectpushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconectpushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>disconect</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelclientConnectToServer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="affaire">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>81</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="essai">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>81</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Grandeur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>id affaire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonModif">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_frequence">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fr√©quence</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_selcEssai">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_idAffaire">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>temp d'aquisition</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tempAqui">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNew">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>nouveau</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_grandeur">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>20</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>slectefile</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>selectFileButtonclicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confDcodage</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>confDecodageFichier()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>connectTCP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconectpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsApplication1Class</receiver>
   <slot>TCPdisconnected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFileButtonclicked()</slot>
  <slot>confDecodageFichier()</slot>
  <slot>connectTCP()</slot>
  <slot>TCPdisconnected()</slot>
 </slots>
</ui>
